modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -263,6 +263,10 @@
 	z-index: 100 !important;
 }
 
+.cke_combopanel.lfr-maximized {
```

Comment:
do we know why this stopped working?

Reply:
I added `lfr-maximized` through liferay-ckeditor here: https://github.com/liferay/liferay-ckeditor/commit/62fe441be4c82545b29a0eb8342139d4613830f1

It checks if the editor is maximized and add or remove the css accordingly.
Please let me know if I answered your question.